### PR TITLE
chore: bump #6872. update node&react sources

### PR DESCRIPTION
**Pull request by davorpa**
## What does this PR do?
Add resource(s) | Improve repo

### Why is this valuable (or not)?

Definitely solves #6872, which was closed by it author in the middle of review

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates. #6872 
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -603,6 +603,7 @@
 
 * [Beginner's Series to: Node.js](https://youtube.com/playlist?list=PLlrxD0HtieHje-_287YJKhY8tDeSItwtg) - Microsoft Developer (YouTube)
 * [Building a RESTful API with Node.js](https://www.youtube.com/playlist?list=PL55RiY5tL51q4D-B63KBnygU6opNPFk_q) - Academind (YouTube)
+* [Data brokering with Node.js: Process data at the speed of technology](https://heynode.com) - Osio Labs Inc. *(signup requested, not required)*
```
Comment (LuigiImVector) [inline courses/free-courses-en.md:606]:
```suggestion
* [Data brokering with Node.js: Process data at the speed of technology](https://heynode.com) - Osio Labs Inc. *(signup required, not requested)*
```

To see the course the signup is required, not requested

**Review by LuigiImVector — APPROVED** [review]

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -603,6 +603,7 @@
 
 * [Beginner's Series to: Node.js](https://youtube.com/playlist?list=PLlrxD0HtieHje-_287YJKhY8tDeSItwtg) - Microsoft Developer (YouTube)
 * [Building a RESTful API with Node.js](https://www.youtube.com/playlist?list=PL55RiY5tL51q4D-B63KBnygU6opNPFk_q) - Academind (YouTube)
+* [Data brokering with Node.js: Process data at the speed of technology](https://heynode.com) - Osio Labs Inc. *(signup required, not requested)*
```
Comment (eshellman) [inline courses/free-courses-en.md:606]: `signup required, not requested` ???

Comment (davorpa) [reply courses/free-courses-en.md:606]: reverting commit feb84cae5b8537eb3c7432a47556cd24e04d8300

Comment (LuigiImVector) [reply courses/free-courses-en.md]: it's obligatory to register

Comment (davorpa) [reply courses/free-courses-en.md]: > it's obligatory to register

You are right, but is also a fact that each chapter of all tutorials has a intro text that is free and doesn't require register to read it. Definitely, if you are interested in contents, you need an user account in the website

Fitting options:
- signup required
- signup requested, not required

Luigi, the last suggestion not is a common resource note reflecting access notes guidelines

**Comment by eshellman** [comment]
Is registration free? if so, this is fine now.

**Comment by davorpa** [comment]
> Is registration free? if so, this is fine now.

Yes!! Using OssioLabs Inc. single sign on (SSO)

**Pull request merged**